### PR TITLE
CudaGraph mixin

### PR DESCRIPTION
**Pull request by grimoire**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by lvhan028** [comment]
may resolve the conflicts

**Review by AllentDan — APPROVED** [review]

File: lmdeploy/pytorch/models/minicpm3.py [hunk 1]
```diff
@@ -395,8 +395,6 @@ def get_input_embeddings(self):
 class MiniCPM3ForCausalLM(nn.Module):
```
Comment (AllentDan) [inline lmdeploy/pytorch/models/minicpm3.py:395]: Lack `CudaGraphMixin`?

File: lmdeploy/pytorch/models/utils/cudagraph.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        input_ids: torch.Tensor,
+        position_ids: torch.Tensor,
+        past_key_values: List[List[torch.Tensor]],
+        attn_metadata: Any = None,
+        inputs_embeds: torch.Tensor = None,
+        **kwargs,
+    ):
+        """return True is model support cudagraph."""
+        seq_lens = input_ids.size(1)
+        return seq_lens <= 256
```
Comment (AllentDan) [inline lmdeploy/pytorch/models/utils/cudagraph.py:52]: What is the motivation here?

File: lmdeploy/pytorch/models/minicpm3.py (hunk 1 above)
Comment (grimoire) [reply lmdeploy/pytorch/models/minicpm3.py:395]: You have set `support_cuda_graph=False`, I thought this model did not support cudagraph.

File: lmdeploy/pytorch/models/utils/cudagraph.py (hunk 2 above)
Comment (grimoire) [reply lmdeploy/pytorch/models/utils/cudagraph.py:52]: graph is captured at pow2 seqlens, input would be padded to ensure static input size.
inputs with large seqlens might have a large amount of unnecessary padding.

**Review by irexyc — APPROVED** [review]

File: lmdeploy/pytorch/models/minicpm3.py (hunk 1 above)
Comment (AllentDan) [reply lmdeploy/pytorch/models/minicpm3.py:395]: It was changed to `True` afterwards.
